scene/2d/tile_map_layer.cpp

```diff
@@ -1636,6 +1597,12 @@ RBSet<TerrainConstraint> TileMapLayer::_get_terrain_constraints_from_painted_cel
 	return constraints;
 }
 
+void TileMapLayer::_tile_set_changed() {
+	emit_signal(CoreStringNames::get_singleton()->changed);
```

Comment:
The order here is different from the others where changed is emitted after the rest, should we unify it?